pyls/plugins/hover.py

```diff
@@ -14,7 +14,10 @@ def pyls_hover(document, position):
     definitions = [d for d in definitions if d.name == word]
 
     if not definitions:
-        # :(
-        return {'contents': ''}
+        return None
 
-    return {'contents': _utils.format_docstring(definitions[0].docstring()) or ""}
+    md_docstring = _utils.format_docstring(definitions[0].docstring())
+    return {'contents': {
+        'type': lsp.MarkupKind.Markdown,
```

Comment:
According to the [spec](https://microsoft.github.io/language-server-protocol/specification) the this should be `'kind': lsp.MarkupKind.Markdown` instead of `'type': lsp.MarkupKind.Markdown` since hover requests return a `MarkupContent` object:

```ts
export interface MarkupContent {
	/**
	 * The type of the Markup
	 */
	kind: MarkupKind;

	/**
	 * The content itself
	 */
	value: string;
}
```